lms/security.py

```diff
@@ -97,8 +97,11 @@ def get_policy(path: str):
             # LTIUser serialized in the state param for the oauth flow
             return LTIUserSecurityPolicy(get_lti_user_from_oauth_callback)
 
-        if path.startswith("/api") and path.endswith("authorize"):
-            # LTUser serialized as query param for authorization failures
+        # LTUser serialized as query param for authorization failures
```

Comment:
We already had this authentication schema, adding a new endpoint to it.

This will read the `authentication` query parameter and decode it as LTIUser.